ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -115,6 +135,21 @@ ColumnLayout {
             markerRoleName: "sourceGroup"
         }
 
+        readonly property var allCollectiblesModel: ConcatModel {
+            sources: [
+                SourceModel {
+                    model: d.communityModel
+                    markerRoleValue: "loadingItemsModel"
+                },
+                SourceModel {
+                    model: d.nonCommunityModel
+                    markerRoleValue: "nonCommunityModel"
+                }
+            ]
+            markerRoleName: "sourceGroup"
+        }
```

Comment:
This looks like something a bit over-complicated. First we split single source model into two and later we concat back together. Probably we could do all transformations on the source except filtering for community/nonCommunity and do that step in a separate two sfpm, one per type at the end of the chain, before passing to grids.

Reply:
As we discussed, this element should be refactored soon.
I will keep it as is since it doesn't have a noticeable impact on performance.